.github/actions/build/action.yml

```
@@ -42,7 +42,7 @@ runs:
       run: |
         mvn --batch-mode --update-snapshots clean compile test-compile
         mvn --batch-mode package -DskipTests -Dmaven.test.skip=true
```

All three `mvn` invocations need the `-Dspotless.check.skip` because the check is part of compile phase:
https://github.com/G-Research/spark-dgraph-connector/actions/runs/9515698290/job/26230466187#step:3:3639
```suggestion
        mvn --batch-mode --update-snapshots -Dspotless.check.skip clean compile test-compile
        mvn --batch-mode -DskipTests -Dmaven.test.skip=true -Dspotless.check.skip package
```